modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef OPENCV_GAPI_RENDER_HPP
+#define OPENCV_GAPI_RENDER_HPP
+
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi/opencv_includes.hpp>
+#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/own/exports.hpp>
+#include <opencv2/gapi/own/scalar.hpp>
+
+namespace cv
+{
+namespace gapi
+{
+
+struct TextEvent
+{
+    std::string text;
+    int x;
+    int y;
+    int ff;
+    double fs;
+    cv::gapi::own::Scalar color;
```

Comment:
May be it make sence to align rith other PRs and introduce GAPI_STANDALONE macro to enable both "part of OpenCV" version which uses native data structures and standalone version with G-API own structures?

Reply:
Replace to cv::Scalar